src/main/java/sk/uniba/fmph/dcs/game_board/ResourceSource.java

```diff
@@ -14,7 +14,7 @@ public final class ResourceSource implements InterfaceFigureLocationInternal {
     private int maxFigureColours;
     private ArrayList<PlayerOrder> figures;
 
-    public ResourceSource(final String name, final Effect resource) {
+    public ResourceSource(final String name, final Effect resource, final CurrentThrow currentThrow, final int playerCount) {
```

Comment:
```suggestion
    public ResourceSource(final String name, final Effect resource, final CurrentThrow currentThrow, final int playerCount, final int maxFigureColours) {
```
Toto tu tiez netreba?